File: 프로그래머스/0/120847. 최댓값 만들기 （1）/최댓값 만들기 （1）.java
class Solution {
    public int solution(int[] numbers) {
        for(int i=0; i < numbers.length - 1; i++) {
            for(int j=i+1; j < numbers.length; j++) {
                if(numbers[i] < numbers[j]) {
                    int temp = numbers[j];
                    numbers[j] = numbers[i];
                    numbers[i] = temp;
                }
            }
        }
        
        return numbers[0] * numbers[1];
    }
}

File: 프로그래머스/0/120847. 최댓값 만들기 （1）/최댓값 만들기 （1）.go
func solution(numbers []int) int {
    for i := 0; i < len(numbers) - 1; i++ {
        for j := i+1; j < len(numbers); j++ {
            if numbers[i] < numbers[j] {
                temp := numbers[j]
                
                numbers[j] = numbers[i]
                numbers[i] = temp
            }
        }
    }

    return numbers[0] * numbers[1];
}